static-site/src/components/ListResources/index.tsx

```diff
@@ -293,14 +296,14 @@ const Tile = ({ tile }: { tile: FilterTile }) => {
 
 
 /**
- * Given a set of user-defined tiles, restrict them to the set of tiles for
- * which the filters are valid given the resources known to the resource listing
- * UI
+ * Given a set of user-defined pathogens, restrict them to the set of pathogens
+ * for which the filters are valid given the resources known to the resource
+ * listing UI
  */
-const useTiles = (tiles?: FilterTile[], groups?: Group[]) => {
-  const [restrictedTiles, setRestrictedTiles] = useState<FilterTile[]>([]);
+const usePathogens = (pathogens: Pathogen[], groups?: Group[]) => {
+  const [restrictedTiles, setRestrictedTiles] = useState<Pathogen[]>([]);
   useEffect(() => {
-    if (!tiles || !groups) return;
+    if (!pathogens || !groups) return;
```

Comment:
The way this is written, `!pathogens` is always going to be false -- you've got `pathogens: Pathogen[]` in the function signature, so at a minimum, there will be an empty array there. 

I'm not sure if the correct fix here is to change the function signature to be `pathogens?: Pathogen[]` (which would more closely match the old version) or change this conditional to be `if (!pathogen.length || !groups)`

Reply:
Good catch – I inadvertently removed the `?`. Added back in [force-push](https://github.com/nextstrain/nextstrain.org/compare/f9a617103fb9f2a22ab0277f94fd1b3649e646df..5c46de7e9269ca8a64bbf3863de9e8b196a91f0a). I don't think the `?` is necessary given existing usage which guarantees a value, but that was already the case before this PR.